src/tags/if_block.rs

```diff
@@ -336,4 +350,75 @@ mod test {
         let output = template.render(&mut context).unwrap();
         assert_eq!(output, Some("fourth".to_owned()));
     }
+
+    #[test]
+    fn contains_comparison() {
+        let text = "{% if \"Star Wars\" contains \"Star\" %}if true{% endif %}";
+        let tokens = compiler::tokenize(&text).unwrap();
+        let template = compiler::parse(&tokens, &options())
+            .map(interpreter::Template::new)
+            .unwrap();
+
+        let mut context = Context::new();
+        let output = template.render(&mut context).unwrap();
+        assert_eq!(output, Some("if true".to_owned()));
+
+        let text = "{% if \"Star Wars\" contains \"Star Trek\"  %}if true{% else %}if false{% endif %}";
+        let tokens = compiler::tokenize(&text).unwrap();
+        let template = compiler::parse(&tokens, &options())
+            .map(interpreter::Template::new)
+            .unwrap();
+
+        let mut context = Context::new();
+        let output = template.render(&mut context).unwrap();
+        assert_eq!(output, Some("if false".to_owned()));
+
+        let text = "{% if movie contains \"Star\"  %}if true{% endif %}";
+        let tokens = compiler::tokenize(&text).unwrap();
+        let template = compiler::parse(&tokens, &options())
+            .map(interpreter::Template::new)
+            .unwrap();
+
+        let mut context = Context::new();
+        context.set_global_val("movie", Value::Str("Star Wars".into()));
+        let output = template.render(&mut context).unwrap();
+        assert_eq!(output, Some("if true".to_owned()));
+
+        let text = "{% if movie contains \"Star\"  %}if true{% else %}if false{% endif %}";
+        let tokens = compiler::tokenize(&text).unwrap();
+        let template = compiler::parse(&tokens, &options())
+            .map(interpreter::Template::new)
+            .unwrap();
+
+        let mut context = Context::new();
+        context.set_global_val("movie", Value::Str("Batman".into()));
+        let output = template.render(&mut context).unwrap();
+        assert_eq!(output, Some("if false".to_owned()));
+    }
+
+    #[test]
+    #[should_panic(expected = "Left-hand side")]
+    fn contains_comparison_non_string_lhs() {
+        let text = "{% if 7 contains \"Star\" %}if true{% endif %}";
+        let tokens = compiler::tokenize(&text).unwrap();
+        let template = compiler::parse(&tokens, &options())
+            .map(interpreter::Template::new)
+            .unwrap();
+
+        let mut context = Context::new();
+        let _output = template.render(&mut context).unwrap();
```

Comment:
Instead of checking for panic, you could

```rust
let output = template.render(&mut context);
assert!(output.is_err());
```
This makes maintenance a little easier because you don't have to worry about string matching the panic message (the error message can change without breaking the tests).

(also applies to the next test in case you decide to implement it)